backend/src/requests/questions/questions.controller.ts

```diff
@@ -68,21 +66,35 @@ export class QuestionsController {
     //@ts-ignore
     return await this.userContentService.getTitleOfQuestion(id);
   }
-  @Get(':id/answers')
+
+  @Get(':id/answers') // TODO returns [], database is not implemented
```

Comment:
The //todo can be removed, the corresponding database services are on the #215 PR, and are currently on review.